errors.js

```diff
@@ -0,0 +1,22 @@
+//psql errors handler
+exports.handlePsqlErrors = (err ,req, res, next) => {
+    if (err.code){
+        res.status(400).send({msg:"Bad request. Please check what you're requesting and try again."})
+    }
+    else next(err);
+}
+    
+//Custom errors handler
+exports.handleCustomErrors = (err ,req, res, next) => {
+    if (err.msg){
+        res.status(err.status).send({msg: err.msg})
+    }
+    else next(err);
+}
```

Comment:
You have no test scenarios that require the `handleCustomErrors` or `handlePsqlErrors` error handlers. If you have no test scenarios for them they shouldn't be put in yet.

Remember it's worth treating your tickets as if you're working alongside other devs. You shouldn't be coding things unrelated to the ticket you're on because it could crossover with another person's ticket.

Reply:
Those are the standard error handlers. so i thouht i'd put them now and now worry about it! :)